app/helpers/search_bar_helper.rb

```diff
@@ -1,32 +1,21 @@
 module SearchBarHelper
-  def mount_search_bar(
-    id,
-    controller: auto_complete_controller_name,
-    url: send("auto_complete_search_#{auto_complete_controller_name}_path"),
-    search_query: params[:search],
-    use_bookmarks: true,
-    use_key_shortcuts: true,
-    autocomplete_id: "searchBar"
-  )
-    bookmarks = {}
-    if use_bookmarks
-      bookmarks = {
-        url: main_app.api_bookmarks_path,
-        canCreate: authorizer.can?(:create_bookmarks),
-        documentationUrl: documentation_url("4.1.5Searching"),
-      }
-    end
-    react_component("SearchBar", {
-                      data: {
-                        controller: controller,
+  def mount_search_bar
+    bookmarks = {
+      url: main_app.api_bookmarks_path,
+      canCreate: authorizer.can?(:create_bookmarks),
+      documentationUrl: documentation_url("4.1.5Searching"),
+    }
+    url = send("auto_complete_search_#{auto_complete_controller_name}_path")
+    react_component("SearchBar", data: {
+                      controller: auto_complete_controller_name,
                       autocomplete: {
-                        id: autocomplete_id,
-                        searchQuery: search_query,
+                        id: 'searchBar',
+                        searchQuery: params[:search],
                         url: url,
-                        useKeyShortcuts: use_key_shortcuts,
+                        useKeyShortcuts: true,
                       },
                       bookmarks: bookmarks,
-                      }})
+                    })
   end
 
   def get_search_props(
```

Comment:
This method seems unused and quite possibly for quite some time. Could we drop it and maybe move the `mount_search_bar` to the `LayoutHelper`?

Reply:
already done, just needed to rebase